.github/workflows/ci.yml

```
@@ -17,6 +17,8 @@ jobs:
       - run: npm ci
       - run: npm run lint
       - run: npm run docs:diff
+      - name: Run Shellcheck
+        run: find . -name '*.sh' -print0 | xargs -0 shellcheck -x
```

Why use `find`?   Is the `-x` option necessary?  (What's it do?)

```suggestion
      - run: shellcheck **/*.sh
```

For that matter, why not use [the shellcheck GH action](https://github.com/marketplace/actions/shellcheck)?

---

The reason I did not use it was because I don't think that's an official Shellcheck project action. Here is the page on shellcheck: https://www.shellcheck.net/wiki/GitHub-Actions. Would be happy to switch, if you want though.

---

Ah, okay.  That seems simple enough.  Maybe just change this to the recommended incantation...
```yaml
    - run: shellcheck **/*.sh
```

I don't think the `find` command is needed.  Nor the `-x` option (what does that even do?)

---

The -x option follows included shellscripts and checks them as well. The find command is so that shellscripts in varying directory levels (including root) will all be found.

---

I can test with the action, but in bash, `echo **/*.sh` appears not to find shellscripts in the immediate directory, only a subdirectory

---

Update tested: https://github.com/codecriticsceo/uuid/actions/runs/12554064345/job/35002278202. I added two bad scripts to a throw-away branch, one in root and one two directories down. `shellcheck **/*.sh` did not find them but the `find` command did. I am going to try updating the shellcheck wiki.

---

Updated the PR with the exact command from the docs (find without -x or xargs)